src/sensors.py

```diff
@@ -0,0 +1,93 @@
+#import BoardInterface
+
+from enum import Enum
+
+# Sensor: Abstract class representing an individual sensor on the test stand.
+# Subclasses will need to override _rawToRealValue method with specific
+# logic to convert the input from the microcontroller board into a sensor measurement.
+
+class Sensor():
+
+    def __init__(self, sensorID, boardID, name):
+        self.sensorID = sensorID
+        self.boardID = boardID
+        self._lastValue = 0.0
+        self.name = name
+
+    def getSensorValue(self):
+        return _lastValue
+
+    def setSensorValue( self, sensorVal ):
+        _lastValue = sensorVal
+
+    def getName(self):
+        return name
+
+    def _rawToRealValue(self, rawValue):
+        pass
+
+    def getSensorID(self):
+        return sensorID
+
+    def getBoardID(self):
+        return boardID
+
+class PressureSensor(Sensor):
+
+    def _rawToRealValue(self, rawValue):         #take in as an int value between 0 and 0xFFF
+        return rawValue * (1500 / 4095)         #Maps from (0x000,0xFFF) to (0,1500) psi
+
+class TempSensor(Sensor):
+
+    def _rawToRealValue(self, rawValue):                 #temperature sensor reading comes in as a 16 bit value. 11 bits for integer part, 5 bits for decimal part
```

Comment:
Try to keep lines to around 80 characters.
Also, comments for functions should also be triple quoted docstrings immediately beneath the function. This makes it much easier to read, and keeps your lines much shorter.

If you are doing a single line comment, its fine to put it on the line if its short, otherwise put it immediately before the line of code in question. There should also be a single space after the # sign.